app/controllers/users_controller.rb

```diff
@@ -0,0 +1,4 @@
+class UsersController < ApplicationController
```

Comment:
in this chapter, we dont use UsersController, therefore, delete it

Reply:
done